lib/main.coffee

```diff
@@ -80,7 +80,11 @@ module.exports =
     not_saved = not item?.buffer?.file?
     no_extension = atom.config.get('wordcount.noextension') && item?.buffer?.file?.path.split('.').length == 1
     current_file_extension = item?.buffer?.file?.path.split('.').pop().toLowerCase()
-    if alwaysOn or no_extension or current_file_extension in extensions or not_saved
+    if atom.workspace.getActivePaneItem() != undefined
```

Comment:
It is better to use CoffeScripts [existential operator](http://coffeescript.org/#operators). This would simplify the definition of `is_buffer` to

```coffee
is_buffer = atom.workspace.getActivePanelItem?().buffer?
```

I am unfamiliar with Atom, but if `atom` or `workspace` could every be undefined, the existential operator can be used to savely access those as well. I.e. `atom?.workspace?.getActivePanelItem?().buffer?`